src/AutoMapper/Configuration/MappingExpression.cs

```diff
@@ -541,7 +541,14 @@ public void Configure(TypeMap typeMap)
                 if (attrs.Any(x => x is IgnoreMapAttribute))
                 {
                     IgnoreDestinationMember(destProperty);
-                    _reverseMap?.ForMember(destProperty.Name, opt => opt.Ignore());
+                    try
+                    {
+                        _reverseMap?.ForMember(destProperty.Name, opt => opt.Ignore());

```

Comment:
Is there a better way to do this than try-catch? Like checking to see if the member exists first?